reana_job_controller/slurmcern_job_manager.py

```diff
@@ -45,6 +47,8 @@ def __init__(
         kerberos=False,
         kubernetes_uid=None,
         unpacked_img=False,
```

Comment:
*suggestion:* Can be removed cause they are not used and `**kwargs` will hold all unused parameters like `kubernetes_uid`, etc.